build.gradle

```diff
@@ -37,18 +26,25 @@ tasks.withType(JavaCompile) {
     options.encoding = "UTF-8"
 }
 
+processResources {
+	inputs.property "version", project.mod_version
+
+	filesMatching("fabric.mod.json") {
+		expand "version": project.mod_version
+	}
+}
 jar {
     manifest {
         attributes([
             "Specification-Title": "bowinfinityfix",
             "Specification-Version": "1",
             "Specification-Vendor": "Parker Young",
             "Implementation-Title": project.name,
-            "Implementation-Version": project.version,
+            "Implementation-Version": project.mod_version,
             "Implementation-Vendor": "Parker8283",
             "Implementation-Timestamp": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ")
         ])
     }
     from "LICENSE"
-    archiveAppendix.set "1.16.x-fabric"
+    archiveAppendix.set "fabric"
```

Comment:
See below comment on versioning

Reply:
This appendix get added only to the -dev.jar, the normal jar is simply named modname-version.jar (so without 1.17.x-fabric).

Reply:
Oh you know what, you're right! Haha, I totally forgot about that. I spent a while trying to get it to work, but just gave up and manually renamed the built jar before uploading it to CF. Good times...alright then this is fine.